### ISSUE TITLE
Unable to use application without pointing device (accessibility problem)

### ISSUE THREAD

**Issue by gulkily** [obsproject/obs-studio#6341]
### Operating System Info

macOS 10.13

### OBS Studio Version

27.1.3

### OBS Studio Log URL

-

### OBS Studio Crash Log URL

-

### Expected Behavior

I expected to be able to access user interface elements using either the Tab key and Enter key, or by using accelerator keys (underlined letters)

Neither option is available, and focus rectangles do not appear at all for most controls.

Without a pointing device, which I am not able to use most of the time, the application is completely inaccessible to me.

Thank you for reading my report. I do not expect any fixes for myself, since I am also using an older OS version, but I hope that you take this matter into consideration for other users.

Best regards,
Your loyal frustrated user.

### Current Behavior

-

### Steps to Reproduce

1.
2.
3.
...
-

**Comment by gxalpha** [comment]
Hello, sorry for the late answer.

Do you have the "Keyboard Controls" enabled in the Accessibility settings of macOS? In theory, that should enable you to browse any application using the keyboard. If you do have this enabled, is it completely impossible to do anything, or are there specific features that don't work?
Also for your info, accelerator keys like on Windows don't exist on macOS.

**Comment by gxalpha** [comment]
As there has been no response, I assume that the steps mentioned above this should fix the problem.
